backend/sozisel/lib/sozisel_web/endpoint.ex

```diff
@@ -11,24 +11,25 @@ defmodule SoziselWeb.Endpoint do
     signing_salt: "z3XWlbLB"
   ]
 
-  socket "/socket", SoziselWeb.UserSocket,
+  socket "/api/socket", SoziselWeb.UserSocket,
     websocket: true,
     longpoll: false
 
-  socket "/live", Phoenix.LiveView.Socket, websocket: [connect_info: [session: @session_options]]
+  socket "/api/live", Phoenix.LiveView.Socket,
+    websocket: [connect_info: [session: @session_options]]
 
   plug Plug.Static,
-    at: "/recording",
+    at: "/api/recording",
```

Comment:
As far as I understand putting the `/api` for socket and graphql endpoint, I don't agree that it should be put for static assets (which are more or less dynamic but still its assets entpoints).